pkg/generators/git_test.go

```diff
@@ -141,12 +141,15 @@ func TestGitGenerateParamsFromDirectories(t *testing.T) {
 		},
 	}
 
-	for _, c := range cases {
-		cc := c
-		t.Run(cc.name, func(t *testing.T) {
+	for _, testCase := range cases {
+		testCaseCopy := testCase
+
+		t.Run(testCaseCopy.name, func(t *testing.T) {
+			t.Parallel()
+
 			argoCDServiceMock := argoCDServiceMock{mock: &mock.Mock{}}
 
-			argoCDServiceMock.mock.On("GetDirectories", mock.Anything, mock.Anything, mock.Anything).Return(c.repoApps, c.repoError)
+			argoCDServiceMock.mock.On("GetDirectories", mock.Anything, mock.Anything, mock.Anything).Return(testCase.repoApps, testCase.repoError)
```

Comment:
make sure you only reference `testCaseCopy` inside this `t.Run` and not `testCase` or you will have unpredictable results running it in parallel